lnrpc/rest-annotations.yaml

```diff
@@ -146,6 +146,9 @@ http:
       delete: "/v1/macaroon/{root_key_id}"
     - selector: lnrpc.Lightning.ListPermissions
       get: "/v1/macaroon/permissions"
+    - selector: lnrpc.Lightning.SendCustomMessage
+      post: "/v1/custommessage/send"
```

Comment:
I think it's enough to remove the `send` prefix here? Given it's a POST message, so you're writing/updating to the endpoint as is.

Reply:
Removed.